snowflake_queries/snowflake_brute_force_ip.yml

```diff
@@ -0,0 +1,16 @@
+AnalysisType: scheduled_rule
+Filename: snowflake_brute_force_ip.py 
+RuleID: Snowflake.BruteForceByIp
+Description: >
+  Detect brute force attacks by monitoring for failed logins
+DisplayName: Snowflake Brute Force Attacks
+Enabled: false
+ScheduledQueries:
+  - Query.Snowflake.BruteForceByIp
+Severity: Medium
+Tests:
+  -
+    Name: Value Returned By Query 
+    ExpectedResult: true
+    Log:
+      Anything: any value
```

Comment:
For the sake of the unit test printing a reasonable `title`, could we have `client_ip` defined here?